src/ec/packages/ec-component-search-form/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@ecl-twig/ec-component-search-form",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1-alpha",
+  "description": "ECL Twig - EC Search Form",
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
as `ec-component-text-input` and `ec-component-button` are used directly in the template, they should be added to dependencies

Reply:
Added in https://github.com/ec-europa/ecl-twig/pull/28/commits/72483ba8d0c2d29290bdd9a64af43fa2ae0d59e7

Reply:
sorry, I meant a dependency to the twig component `@ecl-twig/ec-component-text-input` and `@ecl-twig/ec-component-button`
As they are included in the twig template, if they are not present it will crash. So the dependency is to prevent that.
Also these two should probably be in `dependencies`instead of `devDependencies`